js/modules/k6/grpc/client.go

```diff
@@ -461,7 +461,11 @@ func (c *Client) parseInvokeParams(paramsVal goja.Value) (*invokeParams, error)
 				// TODO(rogchap): Should we manage a string slice?
 				strval, ok := kv.(string)
 				if !ok {
-					return result, fmt.Errorf("metadata %q value must be a string", hk)
+					binval, ok := kv.([]byte)
+					if !ok {
+						return result, fmt.Errorf("metadata %q value must be a string or []byte", hk)
```

Comment:
It seems like this is a user-faced error, so it's better to explicitly refer to the `Uint8Array` instead of the golang's (`[]byte`)